wiki/Tournaments/CMC/2020/en.md

```diff
@@ -0,0 +1,388 @@
+---
+tags:
+  - CMC
+  - CMC 2020
+---
+
+# Cemara Mania Cup \#1 (2020)
+
+![CMC 2020 logo](img/logo.jpg)
+
+The **Cemara Mania Cup \#1 (2020)** (***CMC 2020***) was a double-elimination 2v2 osu!mania 4K tournament hosted by ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) and co-hosted by ![][flag_ID] [LLENN-](https://osu.ppy.sh/users/7428653). The tournament was open to all osu!mania players from Indonesia with a [global osu!mania 4K rank](https://osu.ppy.sh/rankings/mania/performance?variant=4k) of #4000 or lower. It was the 1st iteration of the Cemara Mania Cup.
```

Comment:
```suggestion
The **Cemara Mania Cup \#1 (2020)** (***CMC 2020***) was a double-elimination 2v2 osu!mania 4K tournament hosted by ![][flag_ID] [-AMIYA-](https://osu.ppy.sh/users/7219949) and co-hosted by ![][flag_ID] [LLENN-](https://osu.ppy.sh/users/7428653). The tournament was open to all osu!mania players from Indonesia with a [global osu!mania 4K rank](https://osu.ppy.sh/rankings/mania/performance?variant=4k) of #4000 or lower. It was the first iteration of the Cemara Mania Cup.
```
not sure if this was an intended spelling for tourney articles; ignore if so

Reply:
i think both are applicable (i've seen that some articles are using cardinal numbers and some others are using ordinal numbers instead) but yeah why not \:P

![image](https://user-images.githubusercontent.com/36564236/103252011-b3399f80-49ad-11eb-9a38-8af7c4d94d12.png)

![image](https://user-images.githubusercontent.com/36564236/103252035-bfbdf800-49ad-11eb-8dc4-055a63721919.png)